configuration/pih/liquibase/liquibase-haiti.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- This liquibase file is run during startup *before* metadata is loaded -->
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+    <changeSet id="20210916-clear-location-tag-maps" author="mgoodrich">
+
+        <comment>
+            UHM-5770: Refactor location tag configuration into Iniz locationtagmaps
+            (One-time clear out of maps so they can be re-initiazed)
+        </comment>
+        <sql>
+            delete from location_tag_map;
+        </sql>
+    </changeSet>
+
+</databaseChangeLog>
```

Comment:
Per my comment on the other PR, can we avoid this?  It looks like you are defining a large superset of tags in each file, shouldn't this mean that iniz is deleting from location_tag_map as appropriate?  Are there other tags implicated?